include/rdma/fi_endpoint.h

```diff
@@ -237,7 +237,7 @@ fi_rx_size_left(struct fid_ep *ep)
 	return ep->ops->rx_size_left(ep);
```

Comment:
Er... I was testing various choices and apparently forgot to add this.  I will update.